stl/inc/yvals_core.h

```diff
@@ -1382,10 +1383,11 @@
 #define __cpp_lib_out_ptr 202106L
 #endif // __cpp_lib_concepts
 
-#define __cpp_lib_spanstream      202106L
-#define __cpp_lib_stdatomic_h     202011L
-#define __cpp_lib_string_contains 202011L
-#define __cpp_lib_to_underlying   202102L
+#define __cpp_lib_ranges_starts_ends_with 202106L

```

Comment:
We should probably move this into the previous `#ifdef __cpp_lib_concepts` block..